packages/www/pages/docs/guides/debugging-guide.mdx

```diff
@@ -0,0 +1,72 @@
+import DocsLayout from "../../../components/DocsLayout";
+
+<DocsLayout>
+
+
+# Debugging Livepeer.com Stream Playback
+
+This step-by-step guide will help you debug common stream playback problems. Try these steps before reaching out to Livepeer.com directly.
+
+We will use the [Livepeer.com test player](/app/test-player) to debug. You must be logged into the Livepeer.com dashboard. 
+
+#### Prerequisite: Locate your playback URL. 
+
+Your playback URL is specific to the stream ingest region. 
+- This [Livepeer.com API guide](/docs/guides/api/playback-a-live-stream) touches on finding your playback URL by combining a base URL with a `playbackId`. If you don't know your stream `playbackId`, you can retrieve it via [`GET /stream/<id>`](/docs/api/stream/get-stream).
+- You can also find the playback URL on the Livepeer.com dashboard stream page, `https://livepeer.com/app/stream/<stream-id>`. Learn more [here](/docs/guides/dashboard/playback-a-stream).
+
+## Step 1
+
+Load your playback URL in the [Livepeer.com test player](/app/test-player). If you get a "Stream not found" error, confirm that your URL is correct. It should look something like this:  https://region-cdn.livepeer.com/hls/123456abcdef7890/index.m3u8
+
+## Step 2
+
+Does anything playback in the player of *your* choice?
```

Comment:
Even though *your* is italicized here, I still missed the distinction here the first time that this was asking me to compare the lp test player to another player, as the previous step directed me to the lp player and to try to play it there. I wonder if this is better as a question like..."What seems to be the problem you are experiencing?" 

* The video is not playing in your application at all. -> Step 3
* The video is playing, but the quality is poor. -> Step 5
* I'm not sure. -> Step 4

Take it or leave it :)

Reply:
If you are confused by the wording, I'm sure other will be. I'll implement the suggested text change.